src/App.tsx

```diff
@@ -1,9 +1,15 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, queryChange] = useState('');
+
+  function handleChange(value: string) {
+    queryChange(value);
```

Comment:
here is better place to make query lower case instead of props for MoviesList component